torch/testing/_internal/common_methods_invocations.py

```diff
@@ -485,7 +486,7 @@ def __init__(self,
                  sample_inputs_func=None,  # function to generate sample inputs
 
                  # the following metadata relates to dtype support and is tested for correctness in test_ops.py
-                 dtypes=floating_types(),  # dtypes this function is expected to work with
+                 dtypes=None,  # dtypes this function is expected to work with
                  # the following dtypesIf... options override the dtypes value
                  # on their respective device types
                  dtypesIfCPU=None,  # dtypes this function is expected to work with on CPU
```

Comment:
Let's update this comment to be "dtypes this function is expected to work with on the CPU, typically unnecessary since it's (now) redundant with the dtypes kwarg above"